global.json

```diff
@@ -1,13 +1,13 @@
 {
   "sdk": {
-    "version": "3.1.300",
+    "version": "6.0.100-preview.6.21355.2",
```

Comment:
Just to confirm, now that .NET 6.0 SDK has been RTM-ed, should we use version `6.0.101` instead of .NET 6.0 Previwq version of `6.0.100-preview.6.21355.2`?